apps/desktop/src/lib/actions/actionService.svelte.ts

```diff
@@ -47,10 +49,12 @@ function injectEndpoints(api: ClientState['backendApi']) {
 				]
 			}),
 			autoBranchChanges: build.mutation<void, { projectId: string; changes: TreeChange[] }>({
-				query: ({ projectId, changes }) => ({
+				extraOptions: {
 					command: 'auto_branch_changes',
-					params: { projectId, changes },
 					actionName: 'Create a branch for the given changes'
+				},
+				query: ({ projectId, changes }) => ({
+					params: { projectId, changes }
 				}),
```

Comment:
All these kinds of arguments can be reduced to `(x)=>x` once all the `build.query()` endpoints have been updated to use extraOptions.